app/ModelSerializers/Summit/Speakers/PresentationSpeakerSerializer.php

```diff
@@ -177,6 +177,42 @@ public function serialize($expand = null, array $fields = [], array $relations =
                         $values['moderated_presentations'] = $moderated_presentations;
                     }
                     break;
+                    case 'accepted_presentations': {
+                        $accepted_presentations = [];
+                        foreach ($speaker->getPublishedRegularPresentations($summit) as $p) {
+                            $accepted_presentations[] = SerializerRegistry::getInstance()->getSerializer($p)->serialize(AbstractSerializer::filterExpandByPrefix($expand, $relation));
+                        }
+                        $moderated_accepted_presentations = [];
+                        foreach ($speaker->getPublishedRegularPresentations($summit, PresentationSpeaker::RoleModerator, true) as $p) {
+                            $moderated_accepted_presentations[] = SerializerRegistry::getInstance()->getSerializer($p)->serialize(AbstractSerializer::filterExpandByPrefix($expand, $relation));
+                        }
+                        $values['accepted_presentations'] = array_merge($accepted_presentations, $moderated_accepted_presentations);
+                    }
+                    break;
+                    case 'alternate_presentations': {
+                        $alternate_presentations = [];
+                        foreach ($speaker->getAlternatePresentations($summit) as $p) {
+                            $alternate_presentations[] = SerializerRegistry::getInstance()->getSerializer($p)->serialize(AbstractSerializer::filterExpandByPrefix($expand, $relation));
+                        }
+                        $moderated_alternate_presentations = [];
+                        foreach ($speaker->getAlternatePresentations($summit, PresentationSpeaker::ROLE_MODERATOR) as $p) {
+                            $moderated_alternate_presentations[] = SerializerRegistry::getInstance()->getSerializer($p)->serialize(AbstractSerializer::filterExpandByPrefix($expand, $relation));
+                        }
+                        $values['alternate_presentations'] = array_merge($alternate_presentations, $moderated_alternate_presentations);
+                    }
+                    break;
+                    case 'rejected_presentations': {
```

Comment:
need to add this on allowed_relations and pass the ids on a non expand

Reply:
done